Training1/airbnb/src/components/Experience/News.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const News = (props) => {
+  return (
```

Comment:
@doankhanh222, We should not break the props into specific prop like that, in the common case we only should pass a prop like newsData: Object

Please try install prop-types npm package and define for each component in the PR